utilities/post_migration.py

```
@@ -1548,6 +1587,11 @@ def check_vms(
             except Exception as exp:
                 res[vm_name].append(f"check_serial_preservation - {str(exp)}")
 
+            try:
+                check_firmware_and_tpm(source_vm=source_vm, destination_vm=destination_vm)
+            except Exception as exp:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)